### PR TITLE
Add AdBlockingRecoveryApp layout and story.

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue:

- #6964.

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js [hunk 1]
```diff
@@ -17,22 +17,93 @@
...
+
 	return (
 		<Fragment>
 			<Header>
 				<HelpMenu />
 			</Header>
 			<div className="googlesitekit-ad-blocking-recovery">
-				Ad Blocking Recovery
+				<div className="googlesitekit-module-page">
+					<Grid>
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: Each grid should have at least one row and cell in it. Maybe we don't need the `Grid` element here?

Comment (eugene-manuilov) [inline assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: I think we can either combine two divs into one or two divs with the grid to reduce nesting:

```js
<div className="googlesitekit-ad-blocking-recovery googlesitekit-module-page">
    <Grid>
```
or
```js
<Grid className="googlesitekit-ad-blocking-recovery googlesitekit-module-page">
```

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js [hunk 2]
```diff
@@ -17,22 +17,93 @@
...
+const { useSelect } = Data;
 
 export default function AdBlockingRecoveryApp() {
+	const breakpoint = useBreakpoint();
+
+	const settingsURL = useSelect(
+		( select ) =>
+			`${ select( CORE_SITE ).getAdminURL(
+				'googlesitekit-settings'
+			) }#/connected-services/adsense`
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:48]: This is a fancy way of getting the settings page URL... Perhaps we should create a separate selector that returns the URL for the settings page with different fragments? Something like `getSettingsPageURL(tab, target, action)`?

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js [hunk 3]
```diff
@@ -17,22 +17,93 @@
...
+											) }
+											fullWidth
+										/>
+									</Cell>
+								</Row>
+							</Grid>
+
+							<Grid className="googlesitekit-ad-blocking-recovery__content">
+								<Row>
+									<Cell mdSize={ 6 } lgSize={ 8 }></Cell>
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:83]: I believe this is where the content of the page will be rendered right? Maybe let's add a placeholder for now?

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @tofumatt. Mostly looks good to me. Added a few comments for you. Please, take a look.

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js (hunk 3 above)
Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:83]: Sure thing. That wasn't in the ACs but I think that's a good point 🙂

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js (hunk 1 above)
Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: Fair enough; I'm copying an existing/similar layout but I'll give that a go 👍🏻

Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: That's probably true, I'll try it out.

Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: Looks like that base `<Grid>` is used to apply the baseline margin to the plugin. We'll need to keep it in here 😓

with: 
CleanShot 2023-05-31 at 15 12 58

without:
CleanShot 2023-05-31 at 15 12 53

Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:60]: We need the one div, but I've combined them 👍🏻

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js (hunk 2 above)
Comment (eugene-manuilov) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js:48]: Thanks, @tofumatt.

File: assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js (hunk 3 above)
Comment (eugene-manuilov) [reply assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js]: Thanks.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
All looks good now, but there is a merge conflict. @tofumatt, could you please fix it?

**Comment by tofumatt** [comment]
Ah, sure thing, sorry I missed it. Fixed. 👍🏻

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
@tofumatt, looks like we also need to add vrt images.

**Comment by tofumatt** [comment]
Ah, thanks, I've pushed them, should be good now 🙂

**Review by eugene-manuilov — APPROVED** [review]

**Pull request merged**